VERSION

```diff
@@ -88,14 +88,14 @@ date="Unreleased developer copy"
 # Version numbers are described in the Libtool current:revision:age
 # format.
 
-libmpi_so_version=60:3:20
+libmpi_so_version=60:4:20
 libmpi_cxx_so_version=60:1:20
 libmpi_mpifh_so_version=60:2:20
 libmpi_usempi_tkr_so_version=60:0:20
 libmpi_usempi_ignore_tkr_so_version=60:0:20
-libmpi_usempif08_so_version=61:0:21
-libopen_rte_so_version=60:3:20
-libopen_pal_so_version=60:3:20
+libmpi_usempif08_so_version=62:0:0
```

Comment:
I do not think that this is right.

Isn't this library ABI compatible with all v4.0.x?  The ABI break in 4.0.3 was a bug, and IIRC, it is the one that accidentally forgot to include some symbols.  So technically, this version _is_ ABI compatible with even the "bad" version of the library (from 4.0.3).

Reply:
Hmmm, yes I see your point.  Then the next question is relative to v4.0.3, this adds new external constants. meaning I should bump the _age_ param.  but relative to v4.0.2 it's not an addition, it fixes it.  I think there's no harm in bumping the age.

Reply:
Are they really new?  Compared to v4.0.2?